core/services/telemetry/client.go

```diff
@@ -0,0 +1,78 @@
+package telemetry
+
+import (
+	"context"
+	"sync/atomic"
+
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/wsrpc"
+)
+
+type Client struct {
+	ctx     context.Context
+	backend TelemetryClient
```

Comment:
Now we have two telemetry clients, one to our internal relay telemetry service and the other to the remote telemetry RPC service.

Unfortunate naming collision here, so just thinking how to be more explicit about this. But ok for now.